src/App.vue

```diff
@@ -1,50 +1,18 @@
 <template>
-  <div class="index">
-    <TheHeader />
-    <TheHero />
-    <MoviesList />
-    <BookingSection />
-    <ScreeningsSection />
+  <div>
+    <TheHeader v-if="!$route.name.startsWith('User')" />
```

Comment:
I like the reasoning behind this and this will work!

However I didn't touch upon that and considered it a bit more advanced - this might be a very nice case to use route meta fields here if you are interested to try them out 👀
They are good when we want to attach some data to our routes which we then can use in many, many clever ways.
[Check it out](https://router.vuejs.org/guide/advanced/meta.html)
If you need any help with this - feel free to reach out! 🙌